modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
+====
 
-== References
+[NOTE]
+====
+The standalone server and managed domain modes differ primarily in how they manage and configure JBoss EAP instances, not in their capabilities.
+====
 
-This section may not possibly have any hands-on....investigate what we can do here....
\ No newline at end of file
+== The Standalone Directory Structure
+
+* After installing a JBoss EAP 8 distribution, the standalone directory initially contains the following subdirectories:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server standalone]$ pwd
+/opt/EAP-8.0.0/standalone
+
+[vagrant@server ~]$ ls
+configuration/
+deployments/
+lib/
+----
+
+* Breif description of these subdirectories:
+
+. **configuration**: This directory holds:
+.. Standalone server configuration files
+.. `standalone_xml_history` subdirectory for version control
+.. `logging.properties` for logger configuration
+.. `mgmt-users.properties` for login credentials
+.. `mgmt-groups.properties` for login roles.
+
+. **deployments**: This directory holds:
+.. Application deployment files used by Jakarta EE, such as EAR, WAR, and JAR files.
+.. Marker files
+
+. **lib**: Contains common JAR files.
+
+* After running JBoss EAP 8 the first time, the following subdirectories of standalone are created:
++
+[subs="+quotes,+macros"]
+----
+data/
+log/
+tmp/
+----
+
+* Breif description of these subdirectories:
+. **data**: A storage location for subsystems handling file system content, like message queues, in-memory databases, or distributed transaction information.
+
+. **log**: The default location for the server log files.
+
+. **tmp**:  For temporary files, like the shared-key mechanism for authenticating local users to the server's command-line interface (CLI).
+
+
+== JBoss EAP Base Directory
+
+* Users can separate the configuration and data directories from the JBoss EAP 8 installation directory, aiding easier upgrades.
+* This allows the binaries to be installed once and run multiple times on the same machine.
+* Two command-line properties enable this:
+
+. **jboss.home.dir**: Installation directory for JBoss EAP binaries.
+. **jboss.server.base.dir**: Base directory for server configuration files.
+
+* You can override runtime properties at the command line by using the `-D` option when starting the server.
+* For instance, when starting the server with standalone.sh:
++
+[subs="+quotes,+macros"]
+----
+$ ./standalone.sh -Djboss.server.base.dir=/path/to/base/directory -Djboss.home.dir=/path/to/home/directory
+----
+
+== Lab: Run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+*Outcome*: In this lab, you should be able to run a standalone JBoss EAP 8 server using a custom location for the server base directory.
```

Comment:
Change to "You should be able to run two instances of JBoss EAP on the same server using....